lib/src/results.rs

```diff
@@ -31,6 +31,33 @@ impl Display for ResultKind {
     }
 }
 
+impl ResultKind {
+    pub fn get(&self, key: &str) -> Value {
+        match self {
+            ResultKind::Item(value) => {
+                if let Some(v) = value.get(key) {
+                    json!((*v).clone())
+                } else {
+                    json!(null)
+                }
+            }
+            ResultKind::List(v) => v //
+                .into_iter()
```

Comment:
```suggestion
                .iter()
```
v == `&Vec` - Calling into_iter() on a reference is the same as doing iter() which is kinda confusing.